src/styles/tailwind.css

```diff
@@ -0,0 +1,5 @@
+@tailwind base;
```

Comment:
I think `styles` should go under `app/styles` as the `app` holds the user facing application

Reply:
right. just moved it.